awx/settings/development.py

```diff
@@ -152,3 +152,8 @@
 # Used for sending commands in automatic restart
 UWSGI_FIFO_LOCATION = '/awxfifo'
 
+try:
+    socket.gethostbyname('docker.for.mac.internal')
+    os.environ['SDB_NOTIFY_HOST'] = 'docker.for.mac.internal'
```

Comment:
This tells the tool where to send the UDP auto-connect packet; you can customize this if you've got some sort of networking that isn't caught by these two cases.